templates/home/dashboard.html

```diff
@@ -208,10 +208,11 @@ <h4 class="text-center mt-3">Volunteers Present</h4>
                         <tr class="text-center">
                             <td class="text-monospace font-weight-lighter">{{ vol_att.volun.roll_no }}</td>
                             <td class="text-monospace font-weight-lighter">
-                                <a href="{% url 'volunteers:profile'  vol_att.volun.profile.user_id%}">
-                                   {{ vol_att.volun.profile.get_full_name }}
-                                </a> 
-                            {% if vol_att.extra %}(E){% endif %}</td>
+                                 <a href="{% url 'volunteers:profile' vol_att.volun.profile.user_id%}" target ="_blank">
+                                   {{ vol_att.volun.profile.get_full_name }}</a>
+                                    {% if vol_att.extra %}(E){% endif %}
```

Comment:
Format it properly like before (ending anchor tag in a separate line).

Reply:
@garg3133 if I put the closing anchor tag in a separate line then there is <space> after the name which shouldn't be included in the link.